game-core/src/test/java/games/strategy/triplea/delegate/battle/steps/FirstStrikeStepOrderTest.java

```diff
@@ -0,0 +1,531 @@
+package games.strategy.triplea.delegate.battle.steps;
```

Comment:
It's really good to see logic being isolated and then tested :+1: 

Such actions helps improve test coverage which is an effective way to detect and lower bug counts. With test coverage in place future changes will be sure to maintain the same contract, which will make those updates easier and safer and not need manual testing. Avoiding manual testing when we simply just re-arrange code but not add new functionality will save us time, and even having a test infrastruture in place makes it eaiser to add new test cases going forward.

All in all, this is a good investment, thank you.